sql/dj_setup_database.in

```diff
@@ -298,6 +298,7 @@ bare-install|install)
 	export DB_FIRST_INSTALL=1
 	symfony_console doctrine:migrations:migrate -n
 	unset DB_FIRST_INSTALL
+	DBUSER=$domjudge_DBUSER PASSWD=$domjudge_PASSWD symfony_console domjudge:load-default-data
```

Comment:
It does show the password in the output of `ps` but since that happens with other commands already in this file I'll accept it.